src/components/pages/Navbar/Navbar.js

```diff
@@ -87,7 +94,24 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
                 role="button"
               />
             </div>
-
+            {isMentor && (
```

Comment:
I like that this a dynamic way of displaying your tool to mentors only.  Nice work!